dataland-frontend/tests/e2e/utils/CompanyUpload.ts

```diff
@@ -25,6 +25,7 @@ export function fillCompanyUploadFields(companyName: string): void {
   cy.get("input[name=headquartersPostalCode]").type("123456", { force: true });
   cy.get("input[name=companyLegalForm]").type("Enterprise Ltd.", { force: true });
   cy.get("input[name=website]").type("www.company.com", { force: true });
+  cy.get("input[name=lei]").type(`LeiValueId:${crypto.randomUUID()}`, { force: true });
```

Comment:
seemingly this does not work: according to the sonar coverage analysis, all identifiers are empty:

![image](https://user-images.githubusercontent.com/47553592/224786181-ffdbc30c-845c-42a2-8eb3-bbaddf6ca198.png)


I would expect if we get this running, then the sonar coverage isn't an issue anymore.

Reply:
ok